contrib/internal/httptrace/httptrace_test.go

```diff
@@ -9,12 +9,18 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"net/url"
+	"os"
+	"strconv"
+	"sync"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/appsec/dyngo/instrumentation"
```

Comment:
A shame this one "leaked out" but I think we should have kept those IP-related functions in `httpsec` in the end, which would have been cleaner to use for you here. We plan to do this small refactoring soon, so we'll adjust this file too when we do.